### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by takara9**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-MYSQL_VERSION = 8.0.28
```
Comment (masa213f) [inline Makefile:1]: I'm not sure. Does this pass the test?

https://github.com/cybozu-go/moco-agent/issues/77

**Review by masa213f — CHANGES_REQUESTED** [review]
This repository does not depend on k8s. Please fix the title of the PR.

File: Makefile (hunk 1 above)
Comment (takara9) [reply Makefile:1]: updated MYSQL_VERSION = 8.0.37

**Review by masa213f — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (shunki-fujita) [reply Makefile:1]: @takara9 @masa213f 
ここも更新しないとCI動かなくないですか？
https://github.com/cybozu-go/moco-agent/blob/09b017017458054f36c89e0eb62672c3d3c8184d/.github/workflows/ci.yaml#L46

Comment (masa213f) [reply Makefile:1]: @shunki-fujita 
Thanks. I update it in https://github.com/cybozu-go/moco-agent/pull/95.

**Pull request merged**